lib/galaxy/jobs/runners/local.py

```diff
@@ -105,7 +105,6 @@ def queue_job(self, job_wrapper):
                                     env=self._environ,
                                     preexec_fn=os.setpgrp)
 
-            proc.terminated_by_shutdown = False
```

Comment:
`subprocess.Popen` doesn't have `terminated_by_shutdown`
and only 1 usage:
`if proc.terminated_by_shutdown:`